test/jira-operations-test.js

```diff
@@ -34,6 +38,47 @@ describe('JIRA Operations Tests', function() {
           parent.fields.issuetype.name.should.eql('Story');
         });
     });
+    it('Find parent from story with Sub-task parent', () => {
+
+      return findParent(dummyJira.issues.LinkedStory1, dummyJira)
+        .then(parent => {
+          parent.fields.issuetype.name.should.eql('Sub-task');
+        });
+    });
+
+    it('Find parent from story with Initiative, Epic, and Sub-task parents', () =>
+    {
+
+      return findParent(dummyJira.issues.LinkedStory2, dummyJira).then( parent =>{
+          parent.fields.issuetype.name.should.eql('Initiative');
+      });
+    });
+
+    it('Find parent from story with Epic and Sub-task parents', () => {
+
+      return findParent(dummyJira.issues.LinkedStory3, dummyJira)
+        .then(parent => {
+          parent.fields.issuetype.name.should.eql('Epic');
+        });
+    });
+
+    it('Find parent\'s parent of a Sub-task that is under a story with a link to a Dispatcher Sub-task', ()=>
+      {
+        return findParent(dummyJira.issues.LinkedSubtask1, dummyJira)
+        .then( subtaskParent =>
+            {
+                return findParent(subtaskParent, dummyJira).then(storyParent=>
+                {
+                    storyParent.fields.issuetype.name.should.eql('Sub-task');
```

Comment:
I think we can remove this `name.should.equal('Sub-task')` assertion. The assertion we really care about is below and I generally prefer to keep my tests as light as possible. Besides, if `storyParent` weren't a subtask I imagine the below assertion would fail regardless.